food/recipes/models.py

```diff
@@ -36,3 +41,74 @@ class Recipe(models.Model):
 
     def __str__(self):
         return self.food_name
+
+    def convert_cook_time(self):
```

Comment:
This could be refactored to take self.cook_time as the argument, then use a dictionary for output.